Changelog.md

```diff
@@ -1,6 +1,7 @@
 ### 0.6.10 (unreleased)
 
 Language Features:
+ * Output compilation error codes.
```

Comment:
I think this is rather a compiler feature.

Reply:
OK.